Formula/pure-ftpd.rb

```diff
@@ -40,6 +40,16 @@ def install
     system "make", "install"
   end
 
+  def caveats
```

Comment:
We prefer not to add new caveats to formulae, since they become outdated quickly and add to maintenance overhead.

Reply:
Removed caveat block.